Dockerfile

```diff
@@ -24,7 +24,7 @@ ENV AIIDALAB_DEFAULT_GIT_BRANCH master
 # Please note the space at the start of the second entry to ensure that the
 # individual entries are whitespace delimited. If no version is provided, it
 # defaults to $AIIDALAB_DEFAULT_GIT_BRANCH.
-ENV AIIDALAB_DEFAULT_APPS "aiidalab-widgets-base@https://github.com/aiidalab/aiidalab-widgets-base@v1.0.0"
+ENV AIIDALAB_DEFAULT_APPS "aiidalab-widgets-base@git+https://github.com/aiidalab/aiidalab-widgets-base@v1.0.0"
```

Comment:
The only thing I would suggest modifying is the comment section above to provide an example that actually works:

```bash
AIIDALAB_DEFAULT_APPS="aiidalab-widgets-base@git+https://github.com/aiidalab/aiidalab-widgets-base@v1.0.0"
AIIDALAB_DEFAULT_APPS+=" quantum-espresso@git+https://github.com/aiidalab/aiidalab-qe@v21.08.0b4"
```